### DISCUSSION TITLE
Can vue support it?

### DISCUSSION THREAD

**Discussion by wwqhy** [rainbow-me/rainbowkit#1819] [Ideas]
Can vue language development be supported?

**Comment by luozygithub** [comment]
Can rainbowkitsupport the vue project?

**Reply by 0xmkh** [reply]
For now we can't support vue since we use a library called wagmi which is react only. Once the wagmi team is going add vue support we'll have a look at is 🙏

**Reply by zorojean** [reply]
Wagmi now supports Vue

**Reply by 0xmkh** [reply]
We're going to take a look, but can't promise that we will support it in the near future.
